src/scripts/constants.ts

```diff
@@ -0,0 +1,299 @@
+export const ufaucetAbi = [
```

Comment:
I would just make this a JSON file its cleaner that way.

Reply:
I want to take advantage of typing this way.

Reply:
JSON imports are typesafe